consensus/core/src/commit_syncer.rs

```diff
@@ -143,14 +144,14 @@ impl<C: NetworkClient> CommitSyncer<C> {
                         // Create range with inclusive start and end.
                         let range_start = prev_end + 1;
                         let range_end = prev_end + inner.context.parameters.commit_sync_batch_size;
-                        // When the condition below is true, [range_start, range_end] contains less number of commits
+                        // When the condition below is true, CommitRange(range_start..=range_end) contains less number of commits
```

Comment:
Revert? The original representation conveys that the start and end are inclusive.